backend/services/table_anomaly_service.py

```
@@ -0,0 +1,433 @@
+            item["is_acknowledged"] = True
+            item["updated_at"] = datetime.utcnow().isoformat()
+            
+            # Cosmos DB에서 업데이트
+            self.db_repository.container.replace_item(item["id"], item)
+            
+            return item
+        except Exception as e:
+            print(f"Error acknowledging anomaly: {e}")
+            raise
+
+    async def get_anomaly_summary(self, tenant_id: str = "default") -> Dict[str, Any]:
```

Ruff (0.13.1)

Unused method argument: `tenant_id`

(ARG002)